resources/api/kio-api.yaml

```
@@ -564,11 +564,10 @@ definitions:
         description: URL of the specification tool
         example: https://github.com/zalando-stups/kio/issues
       criticality_level:
-        description: Criticality level of an application
+        description: Criticality level (tier) of an application
         type: integer
-        minimum: 1 # low
-        maximum: 3 # high
-        default: 2 # medium
+        minimum: 1 # Tier-1
+        maximum: 3 # Tier-3
```

max should be 4 (for not relevant)
```suggestion
        maximum: 4 # not relevant
```